kubernetes/host.go

```diff
@@ -0,0 +1,141 @@
+package kubernetes
+
+import (
+	"fmt"
+	"strings"
+
+	core_v1 "k8s.io/api/core/v1"
+
+	"github.com/kiali/kiali/config"
+)
+
+// Host represents the FQDN format for Istio hostnames
+type Host struct {
+	Service       string
+	Namespace     string
+	Cluster       string
+	CompleteInput bool
```

Comment:
Please, add a comment about when this flag is true, reading the next function is more or less clear, but it can help to others to understand if there is any cornercase missing.